### ISSUE TITLE
Dumping / restoring system collections is confusing/unintuitive

### ISSUE THREAD

**Issue by ropeladder** [arangodb/arangodb#2172]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [X] 3.0

On this operating system:
  - [X] Ubuntu .deb

I'm finding arangodump and arangorestore to have some confusing defaults. First of all, I see no reason why the _graphs and _aqlfunctions aren't included in normal dumps. At least, in `arangodump/restore --help` there should be some mention that you need to use `--include-system-collections true` in order to dump/restore _graphs and _aqlfunctions. Part of what made this confusing for me is that I had no idea that _graphs and _aqlfunctions were stored as "system collections" or what system collections even were.

Another smaller issue I had is that `--include-system-collections` is overridden by `--collections`. I.e. when you are only dumping/restoring certain collections, you have to *also* specify which systems collections you want to specify; if you specify `--collections` and also `--include-system-collections true` but do not specify which systems collections should be dumped/restored (using e.g. `--collections _graph`) they will not be included.

This is further confused by the fact that if you specify system collections using e.g. `--collection _graphs` but do not specify `--include-system-collections true` it will ignore the system collections you specifically included using `--collection`.

**Comment by aquamatthias** [comment]
+1 ran into the same issue and wondered, why all collections are restored from my backup, but not the graph definition.

**Comment by phillamb168** [comment]
Is my issue related to this? When I perform an `ArangoRestore` (on k8s via operator) it brings over all collections, but does NOT bring over the analyzers:
`      errorMessage: "AQL: failure to find arangosearch analyzer with name 'norm_en_lower' while computing result for function 'TOKENS' (while optimizing ast)",
`

**Comment by chdudek** [comment]
I agree, that analyzers and graphs should also be exported when exporting a database. Without the analyzer and graph definition an exported dataset is not usable.
